README.md

```diff
@@ -46,20 +46,14 @@ We're actively trying to make `libsignal-service-rs` fully functional.
 
 If you're looking to contribute or want to ask a question, you're more than welcome to join our development channel on Matrix (#whisperfish:rubdos.be) or Libera.chat (#whisperfish) to get in touch with us!
 
-## Feature flags for libsignal-service
-
-| Feature flag     | Description                                                                                                                                                                             |
-| ---------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `unsend-futures` | This feature removes the `Send` requirement on returned futures. Enabling this flag may be necessary for interoperability with other libraries that don't support `Send` such as actix. |
-
```

Comment:
I'm not sure whether WF will be compatible for now without the Unsend feature.

Reply:
Ah, I can definitely check that. It was my understanding that this was only for `libsignal-service-actix`. I'll double check.

Reply:
with https://github.com/whisperfish/whisperfish/commit/e93ed195c0be420fe34a2b9b965412a58383fde9 I can confirm my initial suspicion, we can remove `unsend-futures`.

```
Checking whisperfish v0.6.0-dev (/Users/gferon/Developer/signal/whisperfish/whisperfish)
Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.52s
```